### PR TITLE
Promise encode images

### PR DESCRIPTION
**Pull request by lasalvavida**
For #99 

Merge after #133

### CODE REVIEW

**Comment by lasalvavida** [comment]
@lilleyse Since #133 is merged now, when you get a chance, could you look at this?

**Comment by lilleyse** [comment]
Sure I'll take a look soon.

File: lib/encodeImages.js [hunk 1]
```diff
@@ -29,23 +37,20 @@ function encodeImages(gltf, callback) {
...
-        callback(gltf);
-    });
+    }
+    return Promise.all(promises);
+}
+
+function loadImageSource(pipelineExtras, mime) {
+    var image = pipelineExtras.jimpImage;
+    if (!defined(image.getBufferAsync)) {
+        image.getBufferAsync = Promise.promisify(image.getBuffer);
```
Comment (lilleyse) [inline lib/encodeImages.js]: It's better to just put this at the top of the file.

**Comment by lasalvavida** [comment]
Updated

**Comment by lilleyse** [comment]
Looks good, that was my only comment.

**Pull request merged**